### DISCUSSION TITLE
在unbuntu18.04系统下面，安装最新版，一直卡在启动界面，是怎么回事了？

### DISCUSSION THREAD

**Discussion by tiger7chan** [electerm/electerm#2562] [Q&A]
![Screenshot from 2022-07-14 10-04-39](https://user-images.githubusercontent.com/16042289/178882043-0874f272-e4d1-45a0-8029-9d027a239dfe.png)

一直在这个界面，完全进不去了

**Comment by zxdong262** [comment]
我用也是有问题，还以为只是我的18.04有问题，我重装了20。04.。。
